src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -589,8 +590,9 @@ function MoneyRequestConfirmationList(props) {
     ]);
 
     const {image: receiptImage, thumbnail: receiptThumbnail} =
-        props.receiptPath && props.receiptFilename ? ReceiptUtils.getThumbnailAndImageURIs(transaction, props.receiptPath, props.receiptFilename) : {};
+        receiptPath && receiptFilename ? ReceiptUtils.getThumbnailAndImageURIs(transaction ?? null, receiptPath, receiptFilename) : ({} as ReceiptUtils.ThumbnailAndImageURI);
```

Comment:
You decided to skip [this comment](https://github.com/Expensify/App/pull/37716#discussion_r1516137171) or is it just lost?

Reply:
I decided to skip it I think its fine to leave it as it was